index.html

```diff
@@ -1,12 +1,245 @@
+                                    <a id="subscribe-text">Subscribe</a>
+                                </div>
+                                <div id="number">
+                                    <p>799</p>
+                                </div>
+                            </div>
+                        </div>
+
+                        <div id="views">
+                            <p id="view">50,500 views</p>
+                        </div>
+                    </div>
+
+                    <hr id="blue-line">
+                    <hr id="hr-line">
+
+                    <div id="share-bar">
+                        <div class="pull-left">
+                            <span><i class="fa fa-plus"></i> Add to</span>
+                            <span><i class="fa fa-share"></i> Share</span>
+                            <span><i class="fa fa-ellipsis-h"></i> More</span>
+                        </div>
+                        <div class="pull-right">
+                            <span><i class="fa fa-thumbs-up"></i> 1234</span>
+                            <span><i class="fa fa-thumbs-down"></i> 456</span>
+                        </div>
+                    </div>
+
+                </section>
+
+                <div id="section2">
+                    <p id="explanation" class="text-black">
+                        <b> Published on Feb 27, 2014</b><br>
+                        How do you contribute to open source? In this video,
+                        I'll describe the easiest way to make a simple contribution
+                        to an open source project -- without using the command line,
+                        a text editor, "forking a repo", or even touching Git outside of Github.
+                        I'll also describe what kinds of contributions open source software projects
+                        are typically looking for. As an example, I use submitting a student solution
+                        to a web development exercise from The Odin Project, a free online curriculum
+                        for learning web development with Ruby on Rails. You can find more information
+                        about that project at http://theodinproject.com.
+
+                    </p>
+                    <hr>
+                    <div id="show-more"><a href="#" class="text-gray">SHOW MORE</a></div>
+                </div>
+                <div id="loading">
+                    <span><i class="fa fa-spinner"></i> loading</span>
+                </div>
+            </main>
+            <aside id="aside">
+                <div id="autoplay" class="sidebar-image">
+                    <span class="pull-left text-black">Up Next</span>
+                    <span class="pull-right text-gray">Autoplay <i class="fa fa-info-circle"></i></span>
+                </div>
+                <section class="sidebar-image">
+                    <img src="/images/pic2.jpg" alt="image" class="thumbnail">
```

Comment:
change those paths to
```
<img src="images/pic2.jpg"
```
 so it can be seen in your [live-demo](https://rawcdn.githack.com/Tripple-A/Youtube-Clone/chore/webpage-layout/index.html)   (remove the first slash)